zerver/openapi/zulip.yaml

```diff
@@ -11339,6 +11352,9 @@ paths:
 
                           Useful for administrative settings UI that allows changing the
                           video call provider.
+
+                          **Changes**: In Zulip 3.0 (feature level 1), added `{"None": 0}` as
+                          an option to disable the video chat provider UI.
```

Comment:
It seemed useful to include the key and value information here, but wasn't sure about how to format it. This is an instance where it doesn't make sense to list out all the potential key/values, but since this one is always present would be good for clients to be aware of.

<details>
<summary>Screenshot of `realm_available_video_chat_providers`</summary>

![Screenshot from 2022-06-23 18-36-56](https://user-images.githubusercontent.com/63245456/175350289-5109e73f-3369-47d9-81ea-29f7d9f6b90d.png)
</details>

Reply:
Hmm, yeah, good find. This seems like it calls for documentation outside the "Changes" section, too -- from the rest of the description there'd be no reason to think that this quirk existed.

Thinking about this a bit more, I think actually the right answer may be that this entry should not be present in this data. It doesn't have the semantics of the rest of the entries. I don't think it's conveying any meaningful information -- it's not like there's going to be a server setting, where on some servers it's mandatory for every realm to have a provider enabled and on other servers it's not and "none" is an option. The client when rendering the relevant settings UI may choose to have some data structure where "none" goes in a list together with each of the server's options, but that's the client's business.

Now on the other hand, whatever endpoint the client is using to *set* this, that should clearly document that 0 (or whatever the right details are) means "no video-call provider". … I'm not actually finding documentation for such an endpoint, though, or for any "set a generic realm setting" endpoint. That may be a gap in our API docs. (One that I'm happy to leave at lowest priority for the short and medium term.)

Reply:
And nit: the text above says "video call", so this should match, even though "video chat" is in itself a fine term.

Reply:
Updated and moved this update to a separate commit. See outstanding question / continued conversation [in this comment](https://github.com/zulip/zulip/pull/22291#discussion_r918075382)